windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,87 @@
+
+## Certificate renewals
+
+Renewals are supported by Certbot for Windows and are even already configured. Indeed the Certbot installer created a scheduled task during the installation to execute `certbot renew` twice a day.
+
+All certificates created with Certbot for Windows are then automatically renewed out-of-the-box, provided that the required environment to renew is available (e.g. credentials, running web server ...). Indeed Certbot remembers the method that you used to prove your control over the domain name at the time you originally created each certificate. This method is also used for automated renewals. Therefore, if your environment has changed since you first obtained your certificate (for example, if TCP port 80 is no longer free), you might need to change the associated authentication method in order for automated renewals to continue working.
+
+## A note on manual hooks
+
+As explained by the [manual plugin documentation](https://certbot.eff.org/docs/using.html#manual), you may need to set up authentication and cleanup hooks to make the `manual` plugin work as expected, in particular with `DNS-01` challenges.
+
+Similarly to Linux, Certbot for Windows supports any file executable for this platform, if it is available in the `PATH`. It can be `.bat`, `.ps1` or even executable binaries like `.exe`.
+
+## A note on files security
+
+Assets generated by Certbot, in particular keys, are critical in terms of security and should never be exposed to unauthorized accesses. Certbot for Windows leverages DACLs implemented by the NTFS filesystem to enforce this control: keys and sensitive materials are readable by default only by the account that runs Certbot, the `Administrators` group and `SYSTEM`.
+
+## Updating Certbot
+
+When a new version of Certbot release is available, please download the new installer for this new version as described in the _Installation_ section, and run it. The installer will update your existing installation of Certbot.
+
+## Getting help, reporting issues, asking for new features
+
+If you encounter some difficulties to install or use Certbot for Windows, there is a dedicated area in the Let's Encrypt Community Forum that can be reached at _[URL to dedicated category on Certbot (beta) for Windows in the Let's Encrypt Community Forum to be defined]_.
+
+If you find an issue, or want to ask for a new feature on Certbot for Windows, do not hesitate to create a new issue in the [Certbot GitHub repository](https://github.com/certbot/certbot/issues). When filing an issue, you can tag it with the `area: windows` tag to ensure that the developers see that it relates to Certbot on Windows. Also please do not forget that some existing features for Linux systems are not yet available on Windows, but are planned to be added (see the section _Supported features_).
```

Comment:
I believe Adrien is correct at https://github.com/certbot/certbot/pull/7492#issuecomment-549583538 and only people with special permissions on our GitHub repo can set these tags.

Reply:
I thought so. Maybe prepending the title with `[Windows]` would be fine in this case. Again, it is to improve the visibility for the support.

Reply:
I don't care that much, but I personally prefer we don't give them any special instructions for opening issues. It complicates the process for the reporter, the first question in our issue template is for their OS, and we should be adding tags like "Windows" while triaging.

Reply:
Ok, I removed the reference to a tag or title suffix.